### ISSUE TITLE
about rot

### ISSUE THREAD

**Issue by shaonian123daly** [TQTQliu/MVSGaussian#38]
你好，作者！我想请问下为什么在计算rot_out时，不用被注释掉的部分，而仅采用归一化

rot_out = torch.ones((B,x.shape[1],4)).to(x.device)
  # rot_out = self.rotation_head(x)
 rot_out = torch.nn.functional.normalize(rot_out, dim=-1)

**Comment by shaonian123daly** [comment]
以及在计算 color_out时，为什么把x和rgb_vr一起cat起来，color_out不是在计算高斯分支的color吗，希望作者能解答一下，万分感谢

**Comment by TQTQliu** [comment]
你好，这是设置都是基于实验结果，因为我们发现固定rotation和学习rotation效果差不多；把x和rgb_vr一起cat起来最后的性能会稍微好一点。
